airbyte-ci/connectors/connectors_insights/pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.poetry]
+name = "connectors-insights"
+version = "0.1.0"
+description = ""
+authors = ["Airbyte <contact@airbyte.io>"]
+readme = "README.md"
+packages = [{ include = "connectors_insights", from = "src" }]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+dagger-io = "0.9.6"
```

Comment:
Why not upgrade at this point?

Reply:
Especially given this is a new project

Reply:
Because all our tools are on this version. If we'd want to run this tool with airbyte-ci we'd have to make sure their version are aligned. I want to be cautious at upgrading dagger as they might ship breaking API changes and I want to make sure everything is globally working fine before bumping the version.